sass/_content.scss

```diff
@@ -5,287 +5,319 @@
 10.1 Posts and pages
 --------------------------------------------------------------*/
 .sticky {
-    background: #333;
+  background: $color-dark-grey-alt;
 
-    .entry-title a {
-        color: #fff;
-        padding-bottom: 2px;
+  .entry-title a,
+  .entry-meta a,
+  .entry-summary a {
+    color: $color-white;
+    padding-bottom: 2px;
 
-        &:hover {
-            border-color: #fff;
-            color: #fff;
-        }
+    &:hover {
+      border-color: transparent;
+      color: $color-white;
     }
+  }
 
-    .entry-subtitle a {
-        color: #aaa;
+  .entry-title a:hover {
+    border-color: $color-white;
+  }
 
-        &:hover {
-            border-color: #999;
-            color: #aaa;
-        }
-    }
+  .entry-meta a,
+  .entry-summary a {
+    border-color: $color-white;
 
-    * {
-        color: #fff;
+    &:hover {
+      border-color: transparent;
     }
+  }
 
-    &:after {
-        display: none;
+  .entry-subtitle a {
+    color: #aaa;
```

Comment:
Color literals like `#aaa` should only be used in variable declarations; they should be referred to via variable everywhere else.